plugins/modules/helm.py

```diff
@@ -537,7 +537,7 @@ def helmdiff_check(
             cmd += " -f=" + values_file
 
     rc, out, err = run_helm(module, cmd)
-    return len(out.strip()) > 0
+    return (len(out.strip()) > 0, out.strip())
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example. 

Also, could you please add a test for this change?

Reply:
> Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Done.

> Also, could you please add a test for this change?

I'll work on this. Is there an example of a helm module test I can use as a base/reference/template?

Thanks.

Reply:
@jfrabaute You can add some tests in https://github.com/ansible-collections/kubernetes.core/blob/main/molecule/default/roles/helm/tasks/tests_helm_diff.yml. What we'd like to test is that when `diff: yes` is specified in the task that the diff is in the output and when `diff: no` is specified, it's not. You may be able to just modify some of the existing tests, but if not that file should work as a guide for adding some new ones.

Reply:
Thanks! I'll work on this.

Reply:
Tests added